app/controllers/decidim/file_authorization_handler/admin/censuses_controller.rb

```diff
@@ -12,6 +12,9 @@ def show
         def create
           authorize! :create, CensusDatum
           if params[:file]
+            if params[:unverify]
+              Authorization.where(organization: current_organization).destroy_all
```

Comment:
Also, this line of code is very dangerous, because Decidim allows having multiple authorizations per organization.

We need to add another condition in the where clause to only remove authorizations of the desired kind.

Reply:
Ok about using a private method. It makes code cleaner.

And I agree about adding a second condition to restrict deleted records